CHANGELOG: Add latest changes of v0.18.0 release (#292)
nspcc-dev/neofs-s3-gw: CHANGELOG.md

This document outlines major changes between releases.

> ## 0.18.0 (16 Dec 2021)

Reviewer: Use format from keepachangelog.com. Then Sami-chan will be able to parse it for auto release.
```suggestion
## [0.18.0]- 2021-12-16
```

### Added

> - Support of MultipartUpload (#186, #187) 

Reviewer 2: Support for

- CORS support (#217)

> - Authmate supports setting of tokens' lifetime in more convenient format (duration) (#258)

Reviewer 3: Maybe something like this would be better:
```suggestion
- Authmate supports setting of tokens lifetime in more convenient format (duration) (#258)
```

Author: Why? We can put multiple tokens (bearer and session) and tokens' lifetime means 'lifetime of tokens'.

Reviewer 3: If I understand correctly `tokens lifetime` means the same as `lifetime of tokens`

Reviewer 4: I don't thinks an apostrophe is necessary here. It's kinda hard to explain, but I don't feel like "tokens" own "lifetime". In the singular form, I would say `a token lifetime` without `'s`-identified possession. Thus, we don't need it in the plural form; so I would prefer: `Authmate supports setting of tokens lifetime in **a** more convenient format`. 
Though, if you doubt, you can always choose the `lifetime of tokens` form.
p.s. I also googled some phrases like "password lifetime" or "account lifetime" and saw no "'s".

> - Generating of a random key for `--no-sign-request` (#276)

Reviewer 2: Generating → Generation

### Changed

> - Bucket name resolving mechanism from listing owners' container to using DNS (#219)

Reviewer 3: Maybe something like this would be better:
```suggestion
- Bucket name resolving mechanism from listing owner containers to using DNS (#219)
```

Author: Typo, but I think it'd be better `owner's containers`
